content/docs/04-clusters/01-public-cloud/01-aws/08-create-cluster.md

```diff
@@ -0,0 +1,193 @@
+
+
+13. Enable or disable node pool taints. If tainting is enabled then you need provide values for the following parameters:
+    
+    |**Parameter**| **Description**|
+    |-------------|---------------|
+    |**Key**      |Custom key for the taint.|
+    |**Value**    | Custom value for the taint key.|
+    | **Effect**  | Make the choice of effect from the drop-down menu. Review the effect table bellow for more details. |
+  
+    #### Effect Table
+    
+    |**Parameter**| **Description**|
+    |-------------|---------------|
+    | **NoSchedule**|  A pod that cannot tolerate the node taint and should not be scheduled to the node. 
+    | **PreferNoSchedule**| The system will avoid placing a non-tolerant pod to the tainted node but is not guaranteed.
+    | **NoExecute**|  New pods will not be scheduled on the node, and existing pods on the node if any on the node will be evicted they do not tolerate the taint. |
+
+14. Click on **Next**.  
+    
+15. The settings page is where you can configure patching schedule, security scans, backup settings, setup role based access control (RBAC), and enable [Palette Virtual Clusters](/devx/palette-virtual-clusters). Review the settings and make changes if needed. Click on **Validate**.
+
+16. Review the settings summary and click on **Finish Configuration** to deploy the cluster. Be aware that provisioning IaaS clusters can take several minutes.
+
+The cluster details page of the cluster contains the status and details of the deployment. Use this page to track the deployment progress.
+
+
+# Validate
+
+You can validate your cluster is up and running by reviewing the cluster details page. Navigate to the left **Main Menu** and click on **Clusters**. The **Clusters** page contains a list of all available clusters managed by Palette. Click on the row for the cluster you wish to review its details page. Ensure the **Cluster Status** field contains the value **Running**.
+
+
+
+
+# Deleting an AWS IaaS Cluster
+
+The deletion of an AWS cluster results in the removal of all instances and associated resources created for the cluster. To perform a cluster deletion, use the following steps. 
+
+
+1. Ensure you are in the correct project scope.
+
+
+2. Navigate to the left **Main Menu** and click on **Clusters**
+
+
+3. Click on the cluster that you want to remove.
+
+
+4. Click on the **Settings** drop-down menu.
+
+
+5. Click on **Delete Cluster**
+
+
+6. Type in the name of the cluster and click on **OK**
+
+The cluster status is updated to **Deleting** while cluster resources are being deleted. Once all resources are successfully deleted, the cluster status is updated to **Deleted** and is removed from the list of clusters.
+
+## Force Delete a Cluster
+
+If a cluster is stuck in the **Deletion** state for a minimum of 15 minutes it becomes eligible for force deletion. You can force delete a cluster from the tenant and project admin scope.
```

Comment:
`If a cluster is stuck in the **Deletion** state for a minimum of 15 minutes, it becomes eligible for force deletion. You can force delete a cluster from the tenant and project admin scope.`